corehq/apps/linked_domain/migrations/0015_add_previews_to_choices_20210617_2010.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.2.24 on 2021-06-17 20:10
+
+from django.db import migrations, models
+
+from corehq.apps.linked_domain.const import MODEL_PREVIEWS, MODEL_FLAGS
+from corehq.apps.linked_domain.models import DomainLinkHistory
+
+
+def _duplicate_toggle_events_into_preview_events(apps, schema_editor):
+    all_toggle_events = DomainLinkHistory.objects.filter(model=MODEL_FLAGS)
+    # iterate through events, copy into a new entry with model='previews'
+    for toggle_event in all_toggle_events:
+        toggle_event.pk = None
+        toggle_event.model = MODEL_PREVIEWS
+        toggle_event.save()
```

Comment:
Want to bring extra attention here. Is this the best way to go about this?

Reply:
yep, this is good